docs/conf.py

```diff
@@ -153,7 +153,7 @@
 ## -- Options for the edit_on_github extension ----------------------------------------
 
 if eval(setup_cfg.get("edit_on_github")):
-    extensions += ["astropy.sphinx.ext.edit_on_github"]
+    extensions += ["sphinx_astropy.ext.edit_on_github"]
```

Comment:
@pllim you just suggested:
> Looks like you need to not use astropy-helpers in docs/conf.py. Instead use https://github.com/astropy/sphinx-astropy
After this change shown here, I no longer get the error that I have been getting (`No module named 'astropy_helpers.sphinx'). I think this is what you were suggesting all along, so if I understood you correctly, then I think you were right.

Reply:
There are also imports in `docs/conf.py`.

Reply:
Right. Fixing that now.

Reply:
Actually, now I'm seeing that all of this work is going to make it easier later to migrate away from the helpers altogether. Previously that seemed like a really big lift, but actually it's really helpful that much of that functionality has been migrated to standalone packages such as sphinx_astropy.

Reply:
I have now eliminated all calls to `astropy_helpers` within `conf.py`, and the docs build now appears to successfully proceed all the way to the final line of `conf.py`